src/libraries/System.Text.RegularExpressions/src/Resources/Strings.resx

```diff
@@ -69,16 +72,16 @@
   <data name="ArgumentNull_ArrayWithNullElements" xml:space="preserve">
     <value>The array cannot contain null elements.</value>
   </data>
-  <data name="BadClassInCharRange" xml:space="preserve">
+  <data name="ShorthandClassInCharacterRange" xml:space="preserve">
     <value>Cannot include class \\{0} in character range.</value>
   </data>
   <data name="BeginIndexNotNegative" xml:space="preserve">
     <value>Start index cannot be less than 0 or greater than input length.</value>
   </data>
-  <data name="CaptureGroupOutOfRange" xml:space="preserve">
+  <data name="QuantifierOrCaptureGroupOutOfRange" xml:space="preserve">
     <value>Capture group numbers must be less than or equal to Int32.MaxValue.</value>
   </data>
-  <data name="CapnumNotZero" xml:space="preserve">
+  <data name="CaptureGroupOfZero" xml:space="preserve">
```

Comment:
Just noticed the [docs ](https://docs.microsoft.com/en-us/dotnet/standard/base-types/regular-expression-language-quick-reference#grouping-constructs) don't mention you can explicitly number a grouped subexpression.